tests/attacks/inference/attribute_inference/test_black_box.py

```diff
@@ -124,15 +124,21 @@ def test_black_box_continuous(art_warning, decision_tree_estimator, get_iris_dat
         inferred_train = attack.infer(x_train_for_attack, pred=x_train_predictions)
         inferred_test = attack.infer(x_test_for_attack, pred=x_test_predictions)
         # check accuracy
-        assert np.allclose(inferred_train, x_train_feature.reshape(1, -1), atol=0.4)
-        assert np.allclose(inferred_test, x_test_feature.reshape(1, -1), atol=0.4)
+        assert (
+            np.count_nonzero(np.isclose(inferred_train, x_train_feature.reshape(1, -1), atol=0.4))
+            > inferred_train.shape[0] * 0.75
```

Comment:
This is just a relatively relaxed way of measuring accuracy for a regression task. Since not 100% of the values are inferred correctly, we check that at least 75% are (within a given distance).